model_evaluation/jigsaw_evaluation_pipeline.ipynb

```diff
@@ -826,106 +812,81 @@
+    "model_bias_analysis.per_subgroup_auc_diff_from_overall(\n",
+    "    test_bias_df, identity_terms_civil_included, MODEL_FAMILIES, squared_error=True, normed_auc=True)"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {
-    "colab_type": "text",
-    "id": "7bEC5cAsyC05"
-   },
+   "metadata": {},
    "source": [
-    "### Pinned AUC Graphs\n",
-    "The graphs below show per-group Pinned AUC for each subgroup and each model. Each\n",
-    "identity group shows 3 points, each representing the pinned AUC for one training \n",
-    "version of the model. More consistency among the values represents less unintended bias."
+    "### General metrics per model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_nick_names = {\n",
+    "    'tf_gru_attention_unbiasN_trainY:v_1537919376': 'normal',\n",
+    "    'tf_gru_attention_unbiasY_trainN:v_1537918796': 'unbiased',\n",
+    "}"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "colab": {
-     "autoexec": {
-      "startup": false,
-      "wait_interval": 0
-     },
-     "base_uri": "https://localhost:8080/",
-     "height": 365
-    },
-    "colab_type": "code",
-    "executionInfo": {
-     "elapsed": 1552,
-     "status": "ok",
-     "timestamp": 1530303491427,
-     "user": {
-      "displayName": "Flavien Prost",
-      "photoUrl": "//lh5.googleusercontent.com/-2GvWuP8dy24/AAAAAAAAAAI/AAAAAAAAAHI/aCatYKxJMXQ/s50-c-k-no/photo.jpg",
-      "userId": "100080410554240838905"
-     },
-     "user_tz": 240
-    },
-    "id": "QqMmPTreOCPf",
-    "outputId": "39cc5e70-b76d-413e-b286-e25f7a68e732"
+    "scrolled": false
    },
    "outputs": [],
    "source": [
-    "pinned_auc_results = model_bias_analysis.per_subgroup_aucs(test_bias_df, identity_terms_civil_included, MODEL_FAMILIES, 'label')\n",
-    "for family in MODEL_FAMILIES:\n",
-    "  name = model_bias_analysis.model_family_name(family)\n",
-    "  model_bias_analysis.per_subgroup_scatterplots(\n",
-    "      pinned_auc_results,\n",
-    "      'subgroup',\n",
-    "      name + '_aucs',\n",
-    "      name + ' Pinned AUC',\n",
-    "      y_lim=(0., 1.0))"
+    "for model_family in MODEL_FAMILIES:\n",
+    "    \n",
+    "    bias_metrics_df = model_bias_analysis.per_subgroup_aucs(\n",
+    "        test_bias_df, identity_terms_civil_included, [model_family], 'label')\n",
+    "    \n",
+    "    ### Make it prettier\n",
+    "    _model = model_family[0]\n",
+    "    _nickname = model_nick_names[_model]\n",
+    "    bias_metrics_df = bias_metrics_df.rename(columns={\n",
+    "        col: col.replace(_model, _nickname) for col in bias_metrics_df.columns\n",
+    "    })\n",
+    "    for col in bias_metrics_df.columns:\n",
+    "        try:\n",
+    "            bias_metrics_df[col] = list(map(lambda x: [round(y,2) for y in x] , bias_metrics_df[col]))\n",
```

Comment:
Could something like this be used instead? 
https://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.round.html

Reply:
I tried it but it does not work. The dataframe returned contains list of values (e.g. df[col][i] = [0.897123]).
df.round(2) works only if df[col][i] = 0.89.